src/PointwiseFunctions/AnalyticData/BnsInitialData/SpecInitialData.hpp

```diff
@@ -0,0 +1,335 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <Exporter.hpp>  // The SpEC Exporter
+#include <limits>
+#include <memory>
+
+#include "DataStructures/CachedTempBuffer.hpp"
+#include "DataStructures/Tensor/EagerMath/DeterminantAndInverse.hpp"
+#include "Elliptic/Systems/BnsInitialData/FirstOrderSystem.hpp"
+#include "Elliptic/Systems/BnsInitialData/Tags.hpp"
+#include "Evolution/NumericInitialData.hpp"
+#include "NumericalAlgorithms/LinearOperators/PartialDerivatives.hpp"
+#include "Options/String.hpp"
+#include "PointwiseFunctions/AnalyticData/BnsInitialData/TovStar.hpp"
+#include "PointwiseFunctions/GeneralRelativity/Tags.hpp"
+#include "PointwiseFunctions/Hydro/EquationsOfState/EquationOfState.hpp"
+#include "PointwiseFunctions/Hydro/InitialData/IrrotationalBns.hpp"
+#include "PointwiseFunctions/Hydro/Tags.hpp"
+#include "PointwiseFunctions/InitialDataUtilities/InitialData.hpp"
+#include "Utilities/Serialization/CharmPupable.hpp"
+#include "Utilities/TMPL.hpp"
+#include "Utilities/TaggedTuple.hpp"
+
+/// \cond
+namespace PUP {
+class er;
+}  // namespace PUP
+/// \endcond
+
+namespace BnsInitialData::InitialData {
+
+/*!
+ * \brief Hydro initial data generated by SpEC.
+ *
+ * This class loads numerical data written out by the SpEC initial data solver.
+ * It uses the `spec::Exporter` linked in from SpEC to interpolate to arbitrary
+ * grid points. The coordinates are assumed to be in SpEC's "grid" frame.
```

Comment:
We should check if SpEC's grid frame is the one that is deformed to the surface of the star